cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/oidc/AuthorizeAction.java

```diff
@@ -209,9 +211,18 @@ private void doOpenIDCodeFlow() throws Exception {
         Subject s = AuthenticationUtil.getCurrentSubject();
         AuthMethod authMethod = AuthenticationUtil.getAuthMethodFromCredentials(s);
         if (authMethod.equals(AuthMethod.ANON)) {
-            
+            // Get the service URL for ac
+            RegistryClient regClient = new RegistryClient();
+            URL regCapabilitiesURL = regClient.getAccessURL(new URI("ivo://cadc.nrc.ca/reg"));
+
+            String regCapURLStr = regCapabilitiesURL.toString();
+            String oidcLoginHostURL = regCapURLStr.replace("/reg/capabilities","");
+
             // send redirect to username/password form
-            StringBuilder redirect = new StringBuilder("oidc-login.html#redirect_uri=");
+            // In future, this reference to /en/login.html can be replaced by
+            // a provide an arbitrary login screen.
```

Comment:
small grammar error

Reply:
changed to:
            // In future, this reference to /en/login.html can be replaced by
            // a mechanism to provide an arbitrary login screen.